server/routes/referrals/referralStartPresenter.ts

```diff
@@ -12,7 +12,7 @@ export default class ReferralStartPresenter {
     return this.draftReferrals
       .sort((a, b) => (new Date(a.createdAt) < new Date(b.createdAt) ? -1 : 1))
       .map(referral => ({
-        id: referral.id.slice(0, 8), // this is totally arbitrary, and probably meaningless to the user
+        serviceUserFullName: `${referral.serviceUser.firstName} ${referral.serviceUser.lastName}`,
```

Comment:
is it worth updating the tests for this?

Reply:
yeh maybe - there aren't any tests right now, and this content is subject to change, so i'm not sure.

Reply:
I'll defer to you on this one, up to you!

Reply:
i'll stick one in. @lawrence-forooghian made a good point last week that often temporary code becomes permanent.